index.bs

```diff
@@ -990,6 +990,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
         Note: A suggested reasonable range for the {{PublicKeyCredentialCreationOptions/timeout}} member of |options| is
         15 seconds to 120 seconds.
 
+        Note: User agent should take cognitive guidelines into considerations regarding timeout for users with special needs.
```

Comment:
Nit: In both cases, either use the plural "User agents" or change to "The user agent" to match the rest of the document.

Otherwise, LGTM.

Reply:
Changed.